libtransmission/transmission.h

```diff
@@ -126,7 +126,7 @@ char const* tr_getDefaultDownloadDir(void);
 #define TR_DEFAULT_RPC_PORT 9091
 #define TR_DEFAULT_RPC_URL_STR "/transmission/"
 #define TR_DEFAULT_PEER_PORT_STR "51413"
-#define TR_DEFAULT_PEER_SOCKET_TOS_STR "default"
+#define TR_DEFAULT_PEER_SOCKET_TOS_STR "cs0"
```

Comment:
```suggestion
#define TR_DEFAULT_PEER_SOCKET_TOS_STR "le"
```
`le` is better suited for bittorrent traffic

Reply:
I changed it to "cs1" in ee098bdf2451da93bc487ea069f64c66e8813e1f rather than "le" to have priority similar to libtorrent which uses 0x20 as default.

Reply:
That's actually a bug in libtorrent, they're using the "QBone scavenger service" DSCP which is defined in [RFC 8622](https://datatracker.ietf.org/doc/html/rfc8622) which has DSCP _codepoint_ 0x01 (accounting for the ECN bits it results in 0x04). The incorrect 0x20 value is obtained if you count the bits backward.

Reply:
updated to `le` in e2a1f9bcefcc57b02b45301d19e4e9d497bb8373